rest-api/test/unit_test/offline_test/biobank_samples_pipeline_mysql_test.py

```diff
@@ -203,11 +279,12 @@ def test_reconciliation_query(self):
                          'Ounconfirmed_sample_4', two_days_ago, two_days_ago)
 
 
-    # On time order and samples from 10 days ago; shows up in rx
+    # On time order and samples from 10 days ago; should not shows up in rx or modified
     p_old_on_time = self._insert_participant(race_codes=[RACE_AIAN_CODE])
     # Old missing samples from 10 days ago don't show up in missing or late.
-    self._insert_order(p_old_on_time, 'OldGoodOrder', BIOBANK_TESTS[:3],
-                       old_order_time, kit_id='kit2')
+    old_on_time_order = self._insert_order(p_old_on_time, 'OldGoodOrder', BIOBANK_TESTS[:3],
+                                           old_order_time, kit_id='kit2')
+    self._modify_order('EDITED', old_on_time_order)
     self._insert_samples(p_old_on_time, BIOBANK_TESTS[:2], ['OldGoodSample1', 'OldGoodSample2'],
```

Comment:
test 'edited' order for more than 10 days, make sure they will not show in reports.